torch/csrc/lazy/core/shape_inference.cpp

```diff
@@ -554,6 +554,14 @@ std::vector<Shape> compute_shape_clamp_min(const at::Tensor & self, const at::Sc
   return {Shape(self.scalar_type(), self.sizes().vec())};
 }
 
+std::vector<Shape> compute_shape_minimum(const at::Tensor & self, const at::Tensor & other) {
```

Comment:
cc @wconstab. 
I'm making minimum/maximum alias of clamp_min/clamp_max so they are not structured anymore and require shape functions. These are the wrong shape functions (should broadcast self and other), so what would be the right shape functions, or how can I let lazy tensor know that minimum and maximum should use clamp_min's shape function?

Reply:
Will might have a better suggestion, but clamp_min/clamp_max just use meta tensors in LTC today, so you could copy what they do:

```
auto out_meta = at::meta::clamp_min(self, other);
return {Shape(out_meta.scalar_type(), out_meta.sizes().vec())};
```

Reply:
oh or maybe an even better option: now that `minimum` and `maximum` are `CompositeExplicitAutograd`, LTC should get them for free (and they'll decompose into clamp_min/clamp_max).

So you can just remove `minimum` and `maximum` from `ts_native_functions.yaml` and let the decomp do the work: https://github.com/pytorch/pytorch/blob/8f5cdc6d5d98f4dfc6d9ef27f43a52957bf1abb9/aten/src/ATen/native/ts_native_functions.yaml#L85